classes/cluster/mk22_lab_advanced/stacklight/init.yml

```diff
@@ -5,14 +5,16 @@ parameters:
     heka_aggregator_host: ${_param:stacklight_monitor_address}
     aggregator_port: 5565
     grafana_user: admin
-    mysql_grafana_password: password
+    grafana_password: password
     grafana_influxdb_host: ${_param:stacklight_monitor_address}
     elasticsearch_port: 9200
     influxdb_stacklight_password: lmapass
+    influxdb_admin_password: password
     influxdb_port: 8086
     influxdb_database: lma
     influxdb_user: lma
     influxdb_password: lmapass
+    kibana_port: 5601
```

Comment:
IIUC ports are hardcoded  now. See #152 
So I think that we can just remove it.